app/resources/Courses.py

```diff
@@ -9,13 +9,12 @@
 class CoursesResource(Resource):
     def get(self):
         courses = {}
-        try:
-            raise ValueError('Manual skip.')
-            courses = Course.query.all()
-            courses = course_schema.dump(courses).data
-        except:
-            with open('./app/JSON/courses.json') as f:
-                courses = json.load(f)
+        # try:
+        courses = Course.query.all()

```

Comment:
I think we should surround this with a try catch in the case that something breaks, and send back a 500